README_SECURITY.md

```diff
@@ -54,17 +73,26 @@ The same by passing an environment variable when launching Docker:
 docker run ... -e extender.authentication.users=file:users/myusers.txt extender/extender;
 ```
 
-Example user file:
+#### Using a URL
+Example using `application.yml` where users are loaded from the content server from `https://www.mysite.com/extender-users`:
 
 ```
-bob = super5ecret,ROLE_WINDOWS,ROLE_LINUX,ROLE_MACOS,enabled
-may = top5ecret,ROLE_MACOS,enabled
+extender:
+    authentication:
+        users: https://www.mysite.com/extender-users
```

Comment:
Perhaps make it clearer that it expects a text file?
In our task definition we use an explicit url to a users.txt.

Reply:
Well it doesn't actually have to be a file. It does an HTTP GET so anything that serves a response in the correct format is fine. It could be a REST API that grabs the users from a DB.

This can perhaps be clarified though.